src/Mapping/Annotation/Slug.php

```diff
@@ -38,6 +38,7 @@ final class Slug implements GedmoAnnotation
     public bool $updatable = true;
     public string $style = 'default'; // or "camel"
     public bool $unique = true;
+    public bool $uniqueOverTranslations = false;
```

Comment:
The new argument needs to be added at the end of the constructor, throwing it in the middle of the list is a B/C break.